crates/turborepo-scm/src/git.rs

```diff
@@ -64,7 +66,38 @@ pub fn changed_files(
     Ok(files)
 }
 
+// Gets the system version of both paths by calling `fs_util::canonicalize`,
+// then strips the `monorepo_root` from the `file_path`.
+fn get_stripped_system_file_path(
+    repo_root: &ProjectRoot,
+    file_path: &Path,
+    monorepo_root: &ProjectRelativePath,
+) -> Result<PathBuf, Error> {
+    // We know the path is relative to the repo root so we can convert it to a
+    // ProjectRelativePath
+    let project_relative_file_path = ProjectRelativePath::new(file_path)?;
+    // Which we then resolve to an absolute path
+    let absolute_file_path = repo_root.resolve(project_relative_file_path);
+    // Then we call canonicalize to get a system path instead of a Unix style path
+    let path = fs_util::canonicalize(absolute_file_path)?;
```

Comment:
Just making sure I understand, where is `file_path` coming from that it's not already a platform-specific path? Is it from user config?

Reply:
It's coming from git, which always returns a unix-style path